src/background.ts

```diff
@@ -0,0 +1,8 @@
+console.log("Hi I'm background script!");
+chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
+  let substring = "https://www.netflix.com/watch";
+  if (changeInfo.url?.includes(substring)) {
+    chrome.tabs.reload(tabId);
```

Comment:
Are we going to reload the page every time a user starts watching a video? Or is this just a first approach to see if the extension actually works? I think a better solution would be to add a listener to the entire netflix.com site and watch to see if the URL changes.

Reply:
In my next commit I fixed the reload issue. Now `content script` is activated when we enter netflix page instead of video page, and when we enter the video, `background script` sends the message to the `content script`, giving the information that it can get the information about the video.